Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -224,6 +255,18 @@ - (void)handleRegisterUserNotificationSettings:(NSNotification *)notification
   _requestPermissionsResolveBlock = nil;
 }
 
+RCT_EXPORT_METHOD(onFinishRemoteNotification:(NSString*)notificationId fetchResult:(UIBackgroundFetchResult)result) {
+  RCTRemoteNotificationCallback completionHandler = self.remoteNotificationCallbacks[notificationId];
+  if (!completionHandler) {
+    RCTLogError([NSString stringWithFormat:
```

Comment:
I think this is the problem - you don't need a `stringWithFormat` inside `RCTLogError`, because `RCTLogError` already takes a first argument with `@`'s in it and does this interpolation. Could you fix this?